### DISCUSSION TITLE
Any plans to update this to Filament V3?

### DISCUSSION THREAD

**Discussion by glaarg-robert** [mailcarrierapp/mailcarrier#17] [General]
Any plans to update this to Filament V3?  I would like to integrate this in to my current Laravel app but it uses Filament V3 so I am unable.

**Comment by danilopolani** [comment]
Hey @glaarg-robert ! There's the plan to upgrade to Filament v3, but it would introduce breaking changes so it should be a major release and - without introducing any other important feature along with the major - right now I'm not pointing in that direction.

Also please note that installing MailCarrier in any already existing Laravel/Filament app can be disruptive since it's born to be a standalone app (it rewrites some login paths, configs etc.)
